### ISSUE TITLE
Remember Device in Authenticator

### ISSUE THREAD

**Issue by bestickley** [aws-amplify/amplify-ui#1271]
### On which framework/platform would you like to see this feature implemented?

React

### Which UI component is this feature-request for?

Authenticator

### Please describe your feature-request in detail.

Easily configure Authenticator to remember device for MFA enabled user pools.

### Please describe a solution you'd like.

Easily configure Authenticator to remember device for MFA enabled user pools.

### We love contributors! Is this something you'd be interested in working on?

- [ ] 👋 I may be able to implement this feature request.
- [ ] ⚠️ This feature might incur a breaking change.

**Comment by nk9** [comment]
This would be great to see. Should it take the form of a simple "Remember this computer" checkbox on the Sign In part of the component? Wherever the checkbox goes, I'd expect it to be automatically enabled if the User Pool has a Remember Devices setting of "User Opt-In".

**Comment by bestickley** [comment]
@nk9, yes, it could show up there, or after the user successfully completed TOTP (or SMS but I haven't tried that MFA config with the new Authenticator yet). I'm not sure how the API calls works right now. But the next time the user shouldn't have to verify with MFA.

**Comment by wlee221** [comment]
This could be generalized to a "remember me" button when you sign in, in conjunction with mfa if needed. We'll need to look into underlying Cognito feature to check the feasibility.

**Comment by reesscot** [comment]
This feature should be possible since it is currently supported in JS API's:
https://docs.amplify.aws/lib/auth/device_features/q/platform/js/

**Comment by quarryman** [comment]
Guys, 3 years since issue reported Authenticator component  still doesn't respect Cognito Device Tracking settings, which might be set to ask user if he wants to remember current device. Can someone from the team elaborate on this? Asking user if he wants to remember device is widely adopted behaviour that is missing
